src/main.rs

```diff
@@ -1148,6 +1161,10 @@ fn run() -> Result<()> {
         commands::tail::start(&target, &user, format, tunnel_port, metrics_port, verbose)?;
     } else if matches.subcommand_matches("login").is_some() {
         commands::login::run()?;
+    } else if let Some(matches) = matches.subcommand_matches("report") {
+        commands::report::run(matches.value_of("log")).map(|_| {
+            eprintln!("Report submission sucessful. Thank you!");
+        })?;
```

Comment:
You should use [`and_then`](https://doc.rust-lang.org/std/result/enum.Result.html#method.and_then) here, I think.

Reply:
in this case, i don't return any result from the combinator - i just want to specify the Ok behavior and then bubble up the Err. `and_then` could prob work too, by returning `Ok(())` after the print? idk I this is alright as-is.

Reply:
Yeah it's fine, I was just in nitpick mode :)